src/main/java/org/apache/lucene/analysis/kr/tagging/Tagger.java

```diff
@@ -21,6 +21,8 @@
 import java.util.Iterator;
 import java.util.List;
 
+import com.google.common.base.Supplier;
+import com.google.common.base.Suppliers;
```

Comment:
I couldn't find it out either, but it is there as compile time dependency when I run maven :/